pyaerocom/data/paths.ini

```diff
@@ -88,8 +88,10 @@ GAWTADsubsetAasEtAl = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/PYAEROCOM/GAWT
 DMS_AMS_CVO = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/PYAEROCOM/DMS_AMS_CVO/data
 GHOST_EEA_DAILY = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/GHOST/data/EEA_AQ_eReporting/daily
 GHOST_EEA_HOURLY = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/GHOST/data/EEA_AQ_eReporting/hourly
+GHOST_EEA_MONTHLY = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/GHOST/data/EEA_AQ_eReporting/monthly
 GHOST_EBAS_DAILY = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/GHOST/data/EBAS/daily
 GHOST_EBAS_HOURLY = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/GHOST/data/EBAS/hourly
+GHOST_EBAS_MONTHLY = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/GHOST/data/EBAS/monthly
 
 [obsnames]
 #used in plots, the file name of the plots
```

Comment:
@hansbrenna you also need to assign an ID for the monthly dataset under [obsnames]

```
[obsnames]

GHOST_EEA_MONTHLY = GHOST.EEA.monthly
GHOST_EBAS_MONTHLY = GHOST.EBAS.monthly
```

This might be the reason why the monthly reading does not work

Reply:
The database was able to find and open the GHOST.monthly data now. 

Reading does not work since the data has time unit `'months since 2018-01-01 00:00:00'` which is not supported by xarray.